src/pages/Welcome/index.js

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
 import { CreateList } from '../../components/create-list';
 import { JoinList } from '../../components/join-list';
-import ContentContainer from '../../components/content-container';
 import Header from '../../components/title';
+import shoppingWelcomeImage from './shopping-welcome-image.svg';
 
 import './style.css';
 
 export const Welcome = () => (
-  <ContentContainer>
+  <main className="welcome-container">
     <Header className="welcome-header">
-      Welcome to your <br />
-      Smart Shopping list!
+      Welcome to your Smart Shopping List!
     </Header>
-    <div className="welcome-form">
-      <CreateList />
-      <p className="welcome-text center">- or -</p>
-      <p className="welcome-text">
-        Join an existing shopping list
-        <br />
-        by entering a three word token.
-      </p>
-      <JoinList />
-    </div>
-  </ContentContainer>
+    <article className="welcome-content">
+      <img
+        className="shoppin-cart-image"
```

Comment:
I'm not sure but I believe that line 16 is `shopping-...` and not `shoppin-...`. hahah 😄